### ISSUE TITLE
1 Unit of Metal gets lost while pouring Metal Ingots

### ISSUE THREAD

**Issue by Angelnoir** [anegostudios/VintageStory-Issues#401]
**Game Version:** 1.12.5
**Platform:** Windows 10
**Modded:** No

### Description
Issue was found by Gryphius218

Smelting 100 Units of Metal and molding it results in a 99 unit Ingot
![image](https://user-images.githubusercontent.com/7362698/75702309-85b09680-5cb5-11ea-9ea6-4b18d72a73a9.png)

### How to reproduce
1. Smelt 100 units of any Metal
2. Pour it into an Ingot Mold
3. 1 Unit sometimes disappears

Not sure how to reproduce it, I myself was not able to pour an uneven amount of units.

**Comment by radfast** [comment]
Hi guys.  I experienced this bug in 1.12.14.

![Ingot 1](https://user-images.githubusercontent.com/6842258/78789530-2c6b0f80-79a5-11ea-97b7-29667dea89e2.jpg)

I have not yet been able to reproduce it but I'll try some more.   I had a 500 unit crucible and poured it quickly into the lamellar moulds you see on the left, then intended to pour the final 100 units into the ingot mould.  The final pour gave me only 98 units for the ingot.  Maybe pouring too quickly meant 2 of the lamellar moulds took 101 units, so there were only 98 units remaining in the crucible? 
 (I did not check the crucible quantity between pours.  Nobody does that!)

------

Then in an attempt to complete the partial ingot, I tried pouring some more tin bronze from a fresh crucible.  Prior to pouring, the ingot mould held 98 units of tin bronze at 780°C - see screenshot.   Pouring from the crucible (using shift-right click as usual) had the effect of **removing** some of the units from the ingot mould.  So the number of units of metal in the ingot mould **decremented**.  I stopped pouring when the ingot mould quantity was 16 units.   The ingot mould temperature did not increase.  The crucible contents did not change - the crucible remained filled.

![Ingot 2](https://user-images.githubusercontent.com/6842258/78789579-3987fe80-79a5-11ea-8578-46eab69abdd5.jpg)

------

When the 16-unit partial ingot was cold, to finish it I made a crucible holding 85 units of fresh tin bronze.  I was able to pour this into the ingot mould _incrementally_ as I would expect.   This worked, but I would expect to have 1 unit remaining in the crucible.   In fact there were 0 units remaining in the crucible.

Is there some kind of rounding effect if the number of units is less than 5?  (I am not sure what I could do with 1 unit crucible, I don't think that can be used for anything.)
